tests/tensorflow/test_sota_checkpoints.py

```diff
@@ -540,18 +546,28 @@ def test_openvino_eval(self, eval_test_struct: EvalRunParamsStruct, sota_checkpo
         exit_code, err_str = self.run_cmd(save_cmd, cwd=PROJECT_ROOT)
 
         if exit_code == 0:
-            mo_cmd = f'{sys.executable} mo.py' \
-                     f' --framework tf' \
-                     f' --input_shape {self.get_input_shape(eval_test_struct.config_name_)}' \
-                     f' --input_model {tf_checkpoint}' \
-                     f' --output_dir {ir_model_folder}'
+            if 'deployment_tools' in MO_DIR.parts:
+                mo_cmd = f'{sys.executable} mo.py' \
+                         f' --framework tf' \
+                         f' --input_shape {self.get_input_shape(eval_test_struct.config_name_)}' \
+                         f' --input_model {tf_checkpoint}' \
+                         f' --output_dir {ir_model_folder}'
+            else:
+                mo_cmd = f'mo' \
+                         f' --framework tf' \
+                         f' --input_shape {self.get_input_shape(eval_test_struct.config_name_)}' \
+                         f' --input_model {tf_checkpoint}' \
+                         f' --output_dir {ir_model_folder}'
```

Comment:
The code is mostly duplicated in the two branches of the conditional.

Reply:
I refactored these pieces of code. Please take a look.